src/builder.rs

```diff
@@ -85,33 +88,53 @@ impl ClientBuilder {
         self
     }
 
-    /// Builds a [`Client`] and checks the dynamodb table is active with the correct schema.
+    /// Sets the grace period after a lease expires before it can be forcefully acquired
+    /// by [`Client::acquire_or_replace_expired_lease`].
+    ///
+    /// This gives the original lease holder extra time to renew the lease if they were
+    /// slightly delayed.
+    ///
+    /// Default `60s`.
+    pub fn grace_period(mut self, grace_period: Duration) -> Self {
```

Comment:
Didn't we discuss this already and there was no need for an additional duration config?